src/main/java/teammates/common/util/EmailType.java

```diff
@@ -13,6 +13,7 @@
     FEEDBACK_CLOSED("TEAMMATES: Feedback session closed [Course: %s][Feedback Session: %s]"),
     FEEDBACK_PUBLISHED("TEAMMATES: Feedback session results published [Course: %s][Feedback Session: %s]"),
     FEEDBACK_UNPUBLISHED("TEAMMATES: Feedback session results unpublished [Course: %s][Feedback Session: %s]"),
+    FEEDBACK_ACCESS_LINKS_RESENT("TEAMMATES: Feedback sessions access links resent."),
```

Comment:
A more appropriate email subject might be: `TEAMMATES: Access links for recent feedback sessions`